sources/packages/api/src/route-controllers/institution/models/institution.dto.ts

```diff
@@ -74,32 +64,56 @@ export class InstitutionDto extends PartialType(CreateInstitutionDto) {
   @IsOptional()
   institutionTypeName: string;
 }
+export class InstitutionAddress {
+  @IsNotEmpty()
+  addressLine1: string;
+  @IsNotEmpty()
```

Comment:
Does addressLine2 also need to be not Empty? Can you please check.

Reply:
Address Line 2 must be optional 😉